cli/deploy-profiling-job.sh

```diff
@@ -0,0 +1,127 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+set -e
+
+# <set_variables>
+export ENDPOINT_NAME="<ENDPOINT_NAME>"
+export DEPLOYMENT_NAME="<DEPLOYMENT_NAME>"
+export PROFILING_TOOL="<PROFILING_TOOL>"
+export COMPUTE_NAME="<COMPUTE_NAME>"
+# </set_variables>
+
+export ENDPOINT_NAME=endpt-`echo $RANDOM`
+export DEPLOYMENT_NAME=blue
+export PROFILING_TOOL=wrk
+export COMPUTE_NAME=profilingTest
+
+# <create_endpoint>
+az ml endpoint create --name $ENDPOINT_NAME -f endpoints/online/managed/simple-flow/1-create-endpoint-with-blue.yml
+# </create_endpoint>
+
+# <check_endpoint_Status>
+az ml endpoint show --name $ENDPOINT_NAME
+# </check_endpoint_Status>
+
+endpoint_status=`az ml endpoint show --name $ENDPOINT_NAME --query "provisioning_state" -o tsv`
+echo $endpoint_status
+if [[ $endpoint_status == "Succeeded" ]]
+then
+  echo "Endpoint created successfully"
+else 
+  echo "Endpoint creation failed"
+  exit 1
+fi
+
+deploy_status=`az ml endpoint show --name $ENDPOINT_NAME --query "deployments[?name=='blue'].provisioning_state" -o tsv`
+echo $deploy_status
+if [[ $deploy_status == "Succeeded" ]]
+then
+  echo "Deployment completed successfully"
+else
+  echo "Deployment failed"
+  exit 1
+fi
+
+# <create_profiling_job_yaml_file>
+# please specify environment variable "IDENTITY_ACCESS_TOKEN" when working with ml compute with no appropriate MSI attached
+tee profiling_job.yml <<EOF
+\$schema: https://azuremlsdk2.blob.core.windows.net/latest/commandJob.schema.json
+command: >
+  entryscript.sh -p {inputs.payload}
```

Comment:
entryscript.sh is a bit consuming, should we rename it? profiling.sh?

Reply:
script name changed to profiling.sh